.github/workflows/AI_Review.yml

```
@@ -0,0 +1,42 @@
+﻿name: CodeRabbit
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)